sonic-net/sonic-mgmt
# [test-gap] Test members in LACP lag

### Description of PR




Summary:
Fixes [#3604](https://github.com/sonic-net/sonic-mgmt/issues/3604)



### Type of change











### Back port request



### Approach
#### What is the motivation for this PR?
Mitigate test gap, verify 16 lag members in LACP is supported.



#### How did you do it?
Created a port channel with 16 members between DUT and PTF, and configure them in the same sub-net.



#### How did you verify/test it?
1. Verify the status of port channel members in DUT is selected as expected.
2. Send ICMP request packet from PTF to DUT and verify ICMP reply packet in PTF
3. Send ICMP request packet from port behind lag to port not behind lag in PTF, and verify packet arrive successfully.



#### Any platform specific information?



#### Supported testbed topology if it's a new test case?
T0 topology



### Documentation
